addons/mail/models/discuss/discuss_gif_favorite.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class GifFavorite(models.Model):
+    _name = 'discuss.gif.favorite'
+    _description = 'Save favorite gif from tenor API'
+
+    tenor_gif_id = fields.Char('Gif id from tenor', required=True)
+
+    _sql_constraints = [
+        ('user_gif_favorite', 'unique(create_uid,tenor_gif_id)', 'User should not have duplicated favorite gif'),
```

Comment:
Wording inconsistency: Gif, gif, GIF...
We need to pick one and always use it in UX strings.

Reply:
should be GIF